Packs/BastilleNetworks/Integrations/BastilleNetworks/BastilleNetworks.yml

```diff
@@ -0,0 +1,338 @@
+category: Network Security
+commonfields:
+  id: Bastille Networks
+  version: -1
+configuration:
```

Comment:
Sorry we forgot to mention this on our call, but you should add the `Trust any certificate (not secure)` and `Use system proxy settings` integration params.